src/components/blog-image.tsx

```diff
@@ -26,6 +26,8 @@ export const BlogImage = ({
         <Image
           src={src}
           alt={alt}
+          placeholder="blur"
+          blurDataURL={src}
```

Comment:
https://nextjs.org/docs/api-reference/next/image#placeholder

> When placeholder="blur", the blurDataURL will be used as the placeholder. If the src is an object from a static import, then blurDataURL will automatically be populated. If the src is a string, then you must provide the blurDataURL property.

I'm not sure we need to add the `blurDataURL` props as we use local images.